presto-hive/src/test/java/com/facebook/presto/hive/AbstractTestHiveFileFormats.java

```diff
@@ -772,18 +772,10 @@ else if (testColumn.getObjectInspector().getTypeName().startsWith("char")) {
                         assertEquals(actualValue, padEnd((String) expectedValue, ((CharType) type).getLength(), ' '), "Wrong value for column " + testColumn.getName());
                     }
                     else if (testColumn.getObjectInspector().getCategory() == Category.PRIMITIVE) {
-                        if (expectedValue instanceof Slice) {
-                            expectedValue = ((Slice) expectedValue).toStringUtf8();
-                        }
-
-                        if (actualValue instanceof Slice) {
-                            actualValue = ((Slice) actualValue).toStringUtf8();
-                        }
```

Comment:
Why is this unreachable ?

Reply:
>actualValue instanceof Slice have already been covered in a previous block on line 745
I don't see this on L745

At L779, we could have expectedValue as a String (converted from Slice on L745), and we must check and reassign actualValue if it is a Slice too ?

My IntelliJ setup is not detecting L779-L781 as dead code btw

Reply:
OK, I'll put it back.